src/mock-data/trainee-profile.ts

```diff
@@ -34,72 +34,87 @@ const mockPersonalDetails: PersonalDetails = {
   detailsNumber: ""
 };
 
+export const mockProgrammeMemberships = [
```

Comment:
is it worth moving this to a spec file so that its omitted from build bundle? but then i suppose you'd have to still import it.

on reval we are using https://www.mocky.io/ for FE api mocks e.g trainee list mock api `https://www.mocky.io/v2/5e997d8a33000062007b2354`

Reply:
I will update the config to ignore these files along with test files.